aten/src/ATen/native/mps/operations/Inverse.mm

```diff
@@ -0,0 +1,82 @@
+#include <ATen/ATen.h>
+#include <ATen/Tensor.h>
+#include <ATen/Utils.h>
+#include <ATen/mps/MPSStream.h>
+#include <ATen/native/mps/OperationUtils.h>
```

Comment:
`#include <ATen/native/mps/OperationUtils.h>` should be the only header needed here (`OperationUtils.h` already imports most of the other headers)